src/main/java/network/brightspots/rcv/CommonDataFormatReader.java

```diff
@@ -196,11 +196,11 @@ void parseXml(List<CastVoteRecord> castVoteRecords)
               Logger.severe("ContestSelection \"%s\" from CVR not found!", contestSelectionId);
               throw new CvrParseException();
             }
-            String candidateId;
+            String candidateName;
```

Comment:
(Not related to this code) Can we get at least one test in here that makes use of the aliases?